packages/plugins/rubygem-foreman_wreckingball/rubygem-foreman_wreckingball.spec

```diff
@@ -22,6 +22,7 @@ Requires: %{?scl_prefix_ruby}ruby(release)
 Requires: %{?scl_prefix_ruby}ruby
 Requires: %{?scl_prefix_ruby}ruby(rubygems)
 Requires: %{?scl_prefix}rubygem(foreman-tasks) >= 0.13.1
+Requires: %{?scl_prefix}rubygem(foreman_puppet)
```

Comment:
I wonder why this isn't generated as a `BuildRequires`, but it's causing the build failure. Did you use [bump_rpm.sh](https://github.com/theforeman/foreman-packaging/blob/rpm/develop/bump_rpm.sh)?

Reply:
The script doesn't work on MacOS and therefore I needed to do the steps manually. I'll have another try..

Reply:
I've regenerated the spec-file using gem2rpm with the default template. Can we retry the build?